src/CMakeLists.txt

```diff
@@ -1,107 +1,78 @@
 # Must be included before CMAKE_INSTALL_INCLUDEDIR is used.
 include(GNUInstallDirs)
-set(${PROJECT_NAME}_installables "")
 
+add_library(secp_precomputed_objs OBJECT EXCLUDE_FROM_ALL
+  precomputed_ecmult.c
+  precomputed_ecmult_gen.c
+)
+
+# Add objects explicitly rather than linking to the object libs to keep them
+# from being exported.
```

Comment:
If you link against it, CMake will complain that it's not added to the export list. If you then add it to that list, it gets [exported as an interface library in the installed `libsecp256k1-*.cmake` files](https://gitlab.kitware.com/cmake/cmake/-/issues/18935#note_521599). I'm unsure of the consequences of that, but because it's so easily avoidable I decided to just avoid finding out. If nothing else it feels a little icky to expose our internal structure when it's not necessary.

I can change it if there's any reason to though.

Reply:
> If you link against it, CMake will complain that it's not added to the export list.

Right. I can reproduced it with `-DBUILD_SHARED_LIBS=OFF`.

Reply:
Ah yes sorry, forgot to mention that requirement.